get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+## DVC Get Started for Experiments
+
+This repository contains a project that trains a classifier for [Fashion
+MNIST][fmnist] dataset in Tensorflow. 
+
+[fmnist]: https://github.com/zalandoresearch/fashion-mnist
+
+It is used as a showcase for [`dvc exp`][gsdvcexp] commands to manage large
+number of experiments. 
+
+[gsdvcexp]: https://dvc.org/doc/start/experiments
+
+After [installing DVC][installdvc] and cloning the repository, you can run:
+
+[installdvc]: https://dvc.org/doc/install
+
+```console
+python -m venv .venv
+. .venv/bin/activate
```

Comment:
[Previous example-get-started](https://github.com/iterative/example-get-started) begins literally from `git clone` and `cd`. 😆 

We even have [an issue](https://github.com/iterative/example-repos-dev/issues/26) regarding to use `python -m venv` instead of `virtualenv` 😄 

I think when someone knows enough about the virtual environments, they know when to skip this advice and if they don't know `venv`, it's a good place to learn. Some members of our audience may not be aware that Python is written to `.py` files and executed by `python`.

Reply:
Agree it's not ideal to be specific about `venv` but what's a better alternative? Just mention it and don't give out any instructions? That wouldn't help beginners. Like @iesahin mentioned advanced users will know how to replace it, probably.

Reply:
If the user has visited a Github repository, IMO we can assume they know how to clone it, and for `cd`, yeah, they are very 

`venv` is Python specific, we can't assume the user knows Python.  It's true that `venv` is related to a particular workflow, cloning to your local machine and experimenting. The user could be cloning this into a container, in that case they should know to skip.

If the user is new to Python and downloaded some earlier/later version of Tensorflow to their machine, we can't be sure that the code will work as expected. 

Asking the user to work in a virtualenv is a better approach, and without `venv`, asking them to run `pip install -r requirements.txt` is dangerous, they may break some other software without knowing, and without `pip install -r requirements.txt`, how can we say them to install the requirements?

I'm assuming it's the user's second day in Python. If it's the second year, they know they can skip this.

Reply:
I don't think this is very critical. But a simple option is to just add a note like in https://dvc.org/doc/install/macos#install-with-pip